docs/how-to/hip_runtime_api/memory_management/unified_memory.rst

```diff
@@ -0,0 +1,737 @@
+.. meta::
+  :description: This chapter describes Unified Memory and shows
+                how to use it in AMD HIP.
+  :keywords: AMD, ROCm, HIP, CUDA, unified memory, unified, memory
+
+.. _unified_memory:
+
+*******************************************************************************
+Unified memory management
+*******************************************************************************
+
+In conventional architectures CPUs and attached devices have their own memory
+space and dedicated physical memory backing it up, e.g. normal RAM for CPUs and
+VRAM on GPUs. This way each device can have physical memory optimized for its
```

Comment:
```suggestion
VRAM on GPUs. This way each device has physical memory optimized for its
```

Reply:
"can" is actually correct here - there are "devices" that share their physical memory with the host (APUs in general, but explicitly the MI300A). But those support unified memory by default